### PR TITLE
Add TioAnime

### PR DESCRIPTION
**Pull request by Zerlingg**
Search:
Discord_9d8laikJQl

Pause:
image

Play:
image

### CODE REVIEW

**Comment by Zerlingg** [comment]
Idk what it failed at, and idk what i should be changing :-)

File: websites/T/TioAnime/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "thumbnail": "https://i.imgur.com/9TgOPi4.png",
+  "color": "#00468b",
+  "tags": [
+    "tioanime",
+    "streaming",
+    "embedded-links",
+    "anime",
+	"tio",
+	"español",
+	"tio anime",
```
Comment (ririxidev) [inline websites/T/TioAnime/dist/metadata.json:24]: no spaces in tags

File: websites/T/TioAnime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+let browsingStamp = Math.floor(Date.now() / 1000);
+let video = {
+    duration: 0,
+    currentTime: 0
+};
+function getTimestamps(videoTime, videoDuration) {
+    const startTime = Date.now();
+    const endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+    return [Math.floor(startTime / 1000), endTime];
+}
```
Comment (ririxidev) [inline websites/T/TioAnime/presence.ts:15]: presence.getTimestamps()

File: websites/T/TioAnime/presence.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+const presence = new Presence({
+    clientId: "815653970445205544"
+});
+let lastPlaybackState = null;
+let playback;
+let browsingStamp = Math.floor(Date.now() / 1000);
+let video = {
```
Comment (ririxidev) [inline websites/T/TioAnime/presence.ts:7]: merge lets together

File: websites/T/TioAnime/presence.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+presence.on("UpdateData", async () => {
+    const presenceData = {
+        largeImageKey: "tioanime"
+    };
+    if (video != null &&
+        document.location.pathname.includes("/ver")) {
+        const videoTitle = document
+            .evaluate("//body//h1[1]", document)
+            .iterateNext();
+        const episode = videoTitle.textContent.split(/(\w+)$/);
```
Comment (ririxidev) [inline websites/T/TioAnime/presence.ts:32]: merge consts

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by Zerlingg** [comment]
I fixed the codacy stuff and it still shows up :|

File: websites/T/TioAnime/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  "service": "TioAnime",
+  "description": {
+    "es": "TioAnime es un sitio web de streaming donde puedes ver anime con subtítulos en español.",
+    "en": "TioAnime is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "TioAnime是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": "tioanime.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Xp4oPgi.jpg",
+  "thumbnail": "https://i.imgur.com/9TgOPi4.png",
```
Comment (Bas950) [inline websites/T/TioAnime/dist/metadata.json:15]: Try and get a less blurry logo and it doesnt look like the logo on the website...

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
oh man, you literally fucked it up. Read docs then we can talk about this.

**Comment by Zerlingg** [comment]
Wdym i fucked it up? Also, what am i supposed to see at the docs? This response seems like i did something bad lol, it would be nice if i could be told what i "fucked up" or what did i do, i wasnt able to respond yesterday because i had exams and forgot, this is just confusing to me

**Comment by ririxidev** [comment]
Look man, check deepscan and 
image
what does that mean? Check this: https://docs.premid.app/en/dev/presence/class#gettimestampsnumber-number Reading docs is better thing.

**Pull request closed without merging**